CoinDock-app/src/App/Auth/Auth.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
I think this file is not being used so can you remove it please

Reply:
Removed